web/src/lib/utils/asset-utils.ts

```diff
@@ -189,7 +189,8 @@ export function getAssetRatio(asset: AssetResponseDto) {
 	let width = asset.exifInfo?.exifImageWidth || 235;
 	const orientation = Number(asset.exifInfo?.orientation);
 	if (orientation) {
-		if (orientation == 6 || orientation == -90) {
+		// 6 - Rotate 90 CW, 8 - Rotate 270 CW
+		if (orientation == 6 || orientation == 8) {
```

Comment:
Sweet. I agree with @uhthomas that a variable name would be better than a comment + magic number. What about enums? Something like

```typescript
enum Rotation {
  CLOCKWISE_90=6,
  CLOCKWISE_270=8,
}
```

Don't necessarily have to do it here, but we could even change the type in the exif info entity and have the response dto be an enum. Then all the clients would know what the values are and mean.

Reply:
Ah, `ffprobe` returns other values for the orientation (-90, 90, etc.). #3023 fixes it.

And we definitely have to refactor `ExifEntity` and `ExifResponseDto` to change the type of the orientation field to enum. Created an issue #3022 for that.